marketplace/cloudinary-assets/src/components/cloudinaryAppConfig/cloudinaryAppConfig.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { AppExtensionSDK } from 'contentful-ui-extensions-sdk';
+
+interface Props {
+  sdk: AppExtensionSDK;
+}
+
+export default class CloudinaryAppConfig extends React.Component<Props> {
+  render() {
+    return <div>Cloudinary App Config</div>;
```

Comment:
Does this need more work?

Reply:
Yes, I'm on it ;)

sorry, you've been autoassigned to the PR. It's WIP, will ping you once done.